DDSim/DDSim/DD4hepSimulation.py

```diff
@@ -108,16 +110,18 @@ def readSteeringFile(self):
         self.__dict__ = obj.__dict__
     self.steeringFile = os.path.abspath(sFileTemp)
 
-  def parseOptions(self):
+  def parseOptions(self, argv=sys.argv):
     """parse the command line options"""
+    self.argv = argv
```

Comment:
This just assigns the "reference" to argv so `self.argv` and `argv` are referring to the same object.
Should do
`self._argv = list(argv)` to avoid surprises later
```
>>> l = []
>>> b = l
>>> b.append("arg")
>>> l
['arg']
>>> c = list(l)
>>> c
['arg']
>>> l.append("arg2")
>>> c
['arg']
>>> l
['arg', 'arg2']
```